### ISSUE TITLE
4.8.0 prometheus exporter broken

### ISSUE THREAD

**Issue by himpierre** [RocketChat/Rocket.Chat#25707]
-->

### Description:

hello devs.

after upgrading to 4.8.0 the prometheus exporter is broken. curling http://hostname:9458/metrics leads to 500 internal server error.

log:
```
TypeError: Value is not a valid number: undefined
    at set (/app/bundle/programs/server/npm/node_modules/prom-client/lib/gauge.js:141:9)
    at Gauge.set (/app/bundle/programs/server/npm/node_modules/prom-client/lib/gauge.js:29:3)
    at app/metrics/server/lib/collectMetrics.js:93:22
    at call (/app/bundle/programs/server/npm/node_modules/connect/index.js:239:7)
    at next (/app/bundle/programs/server/npm/node_modules/connect/index.js:183:5)
    at Function.handle (/app/bundle/programs/server/npm/node_modules/connect/index.js:186:3)
    at Server.app (/app/bundle/programs/server/npm/node_modules/connect/index.js:51:37)
    at Server.emit (events.js:400:28)
    at Server.emit (domain.js:475:12)
    at parserOnIncoming (_http_server.js:900:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
```

### Steps to reproduce:

1. upgrade to 4.8.0
2. curl http://rocketchatserver:9458/metrics
3. internal server error

### Expected behavior:

prometheus exporter should work.

### Actual behavior:

internal server error

### Server Setup Information:

- Version of Rocket.Chat Server: 4.8.0
- Operating System: linux
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: yes
- NodeJS Version: 14
- MongoDB Version: 4.4

**Comment by ankar84** [comment]
We confirm that issue on our deployment.

**Comment by ankar84** [comment]
Looks like fixed in 4.8.1
@himpierre can you check to be sure?

**Comment by ankar84** [comment]
fixed in 4.8.1 via https://github.com/RocketChat/Rocket.Chat/pull/25781
